common/reactive-grpc-common/pom.xml

```diff
@@ -18,6 +18,7 @@
     <modelVersion>4.0.0</modelVersion>
 
     <artifactId>reactive-grpc-common</artifactId>
+    <packaging>bundle</packaging>
```

Comment:
Won't this change the Maven coordinates for the artifacts? Can the same thing be accomplished without changing the packaging?

Reply:
The Maven coordinates (groupId, artifactId, version) will stay the same. The artifact is still just packaged as a jar file, which for non-osgi projects is the same as a regular jar.
The only difference is that the MANIFEST.MF is enriched with the following headers:
```
Bundle-Description: Integrating gRPC with reactive technologies.
Bundle-License: https://github.com/salesforce/grpc-java-contrib/blob/mas
 ter/LICENSE
Bundle-ManifestVersion: 2
Bundle-Name: reactive-grpc-common
Bundle-SymbolicName: com.salesforce.servicelibs.reactive-grpc-common
Bundle-Version: 0.9.1.SNAPSHOT
Created-By: Apache Maven Bundle Plugin
Export-Package: com.salesforce.reactivegrpc.common;version="0.9.1";uses:
 ="io.grpc.stub,org.reactivestreams"
Import-Package: io.grpc,io.grpc.stub,org.reactivestreams;version="[1.0,2
 )",com.google.common.base;version="[20.0,21)"
Require-Capability: osgi.ee;filter:="(&(osgi.ee=JavaSE)(version=1.6))"
Tool: Bnd-2.3.0.201405100607
```

Reply:
I've updated the PR. Using the manifest goal of the maven-bundle-plugin to generate the bundle manifest. The maven-jar-plugin is used to add the generated manifest to the final artifact.

The Maven coordinates will remain the same, as well as the packaging type.